lib/output/utils.js

```diff
@@ -26,6 +34,9 @@ const ctorRegistrySymbol = Symbol.for("[webidl2js]  constructor registry");
 // This only contains the intrinsic names that are referenced from the `ctorRegistry`:
 const intrinsicConstructors = ["Array"];
 
+const IteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf([][Symbol.iterator]()));
+const AsyncIteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf(async function* () {}).prototype);
```

Comment:
These exports could be removed now; should they be?

Reply:
I think removing them would technically be a major bump. Maybe add a comment here saying we should remove them when we bump?

Reply:
Makes sense. It's worth noting though that there's already a _bit_ of incompatibility between the main branch and the last release; interfaces generated after #234 expect `initCtorRegistry` to be the only way of initializing the `ctorRegistry`, while code generated before #234 naturally doesn't call `initCtorRegistry`, as it didn't exist. Installing pre-#234 interfaces before post-#234 interfaces causes the latter to throw, because the `ctorRegistry` was created but `%IteratorPrototype%` was not captured. There's a workaround (calling `initCtorRegistry` manually before installing interfaces), which I've been using in local testing, but it's not documented.

I sort of assumed that the next release would be major for this reason, though admittedly that would probably be more of a pain for jsdom than just using the workaround. I'm not terribly clear on what the semver guarantees mean for codegen projects, much less for `domexception` and other projects whose public interface is only _part_ codegen.